x-pack/plugins/security_solution/public/management/cypress/tasks/create_endpoint_host.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { CreateAndEnrollEndpointHostResponse } from '../../../../scripts/endpoint/common/endpoint_host_services';
+
+export const createEndpointHost = (
```

Comment:
Just FYI: the `cy.task('createEndpointHost')` task is currently only registered for the "real" endpoint tests, so this task can not be used in the mocked data suite. I initially did not create a helper function because of that, but I'm not also not against having it (and like it better because it's easier to use than `cy.task`)